ci/test/00_setup_env_native_qt5.sh

```diff
@@ -7,13 +7,13 @@
 export LC_ALL=C.UTF-8
 
 export CONTAINER_NAME=ci_native_qt5
-export DOCKER_NAME_TAG=ubuntu:18.04  # Check that bionic gcc-7 can compile our c++17 and run our functional tests in python3, see doc/dependencies.md
-export PACKAGES="python3-zmq qtbase5-dev qttools5-dev-tools libdbus-1-dev libharfbuzz-dev"
+export DOCKER_NAME_TAG=ubuntu:18.04  # Check that bionic gcc-8 can compile our C++17 and run our functional tests in python3, see doc/dependencies.md
```

Comment:
```suggestion
export DOCKER_NAME_TAG=ubuntu:18.04  # Check that bionic gcc-8 (with libstdc++-8) can compile our C++17 and run our functional tests in python3, see doc/dependencies.md
```

Reply:
If you're compiling using GCC, using `libstdc++` of the same version should be implied. I don't even think you can use libc++ with GCC?

Reply:
Fair enough